src/models/Puck.js

```diff
@@ -1,23 +1,62 @@
 import ElectricCharge from './ElectricCharge';
-import { ELECTRIC_CHARGE_TYPE } from '../const/charge.const';
-import { PUCK_VELOCITY_DIVIDER } from '../const/puck.const';
+import { CHARGE_SIZE, ELECTRIC_CHARGE_TYPE } from '../const/charge.const';
+import {
+  PUCK_POSITION,
+  PUCK_RADIUS,
+  PUCK_VELOCITY_DIVIDER,
+} from '../const/puck.const';
 
 export default class Puck extends ElectricCharge {
   constructor(x, y) {
     super(x, y, ELECTRIC_CHARGE_TYPE.POSITIVE);
     this.velocity = { x: 0, y: 0 };
     this.acceleration = { x: 0, y: 0 };
+    this.radius = PUCK_RADIUS;
+    this.previousPosition = { x: [], y: [] };
```

Comment:
It would be more efficient to store previous positions as an array of objects
```js
this.previousPositions = [];

// storing position:
this.previousPositions.push({ x: 100, y: 100 });
```